src/templates/react/Content/src/App.fs

```diff
@@ -66,11 +66,15 @@ let root model dispatch =
                     [ pageHtml model.currentPage ] ] ] ] ]
 
 open Elmish.React
+#if DEBUG
 open Elmish.Debug
```

Comment:
@alfonsogarciacaro correct me if I'm wrong, but I think webpack will eliminate dead code, so opening a namespace is harmless unless one uses something from it.